frontend/components/charts/CumulativeSignedExtrinsicsChart.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <div>
+    <div v-if="loading" class="text-center py-4">
+      <Loading />
+    </div>
+    <div v-else>
+      <ChartFilter :buttons="filterButtons" :active-button="activeButton" />
+      <ReactiveLineChart
+        id="line"
+        :chart-data="chartData"
+        :options="chartOptions"
+        :height="200"
+      />
+    </div>
+  </div>
+</template>
+<script>
+import ReactiveLineChart from '@/components/charts/ReactiveLineChart.js'
+import Loading from '@/components/Loading.vue'
+import ChartFilter from '@/components/ChartFilter.vue'
+import signedExtrinsic from '@/mixins/signedExtrinsic.js'
+
+export default {
+  components: {
+    ReactiveLineChart,
+    Loading,
+    ChartFilter,
+  },
+  mixins: [signedExtrinsic],
+  data() {
+    return {
+      filterButtons: [
```

Comment:
Let's move these buttons as default ones to the filter component.

Reply:
Moved to mixing